src/data/pokemon/species_info/gen_1.h

```diff
@@ -10184,11 +10178,18 @@ const struct SpeciesInfo gSpeciesInfoGen1[] =
         .trainerScale = 256,                                \
         .trainerOffset = 0,                                 \
         .footprint = gMonFootprint_Tauros,                  \
-        .formSpeciesIdTable = sTaurosFormSpeciesIdTable
+        .formSpeciesIdTable = sTaurosFormSpeciesIdTable     \
 
     [SPECIES_TAUROS] =
     {
         TAUROS_MISC_INFO,
+        .baseHP        = 75,                                \
+        .baseAttack    = 100,                               \
+        .baseDefense   = 95,                                \
+        .baseSpeed     = 110,                               \
+        .baseSpAttack  = 40,                                \
+        .baseSpDefense = 70,    
+        .baseSpDefense = 70,       
```

Comment:
Okay, that's also on me. Punished for not checking thoroughly. You don't need the backslashes here. They are needed for macros but not for normal data. 

I hope this is all :skull:

Reply:
lmao im so sorry i cant compile first im on my friends laptop